src/main/java/no/finn/unleash/DefaultUnleash.java

```diff
@@ -82,6 +82,8 @@ public boolean isEnabled(final String toggleName, final UnleashContext context ,
             enabled = defaultSetting;
         } else if(!featureToggle.isEnabled()) {
             enabled = false;
+        } else if(featureToggle.getStrategies().size() == 0) {
```

Comment:
The client did not handle the case of empty strategies correctly. Discovered with the help of the client-specifications :+1: